packages/terra-toolkit/src/spectre.js

```diff
@@ -0,0 +1,73 @@
+import http from 'http';
+import request from 'request';
+import fs from 'fs';
+
+exports.createSpectreRun = (spectreRunMap, suiteName) => () => new Promise((resolve, reject) => {
+  const returnedSpectreRunMap = spectreRunMap;
+
+  function responseCallback(res) {
+    res.setEncoding('utf8');
+    res.on('data', (responseData) => {
+      returnedSpectreRunMap[suiteName] = JSON.parse(responseData).id;
+    });
+    res.on('end', () => {
+      resolve(returnedSpectreRunMap);
+    });
+  }
+
+  if (process.env.SAVE_TO_SPECTRE === 'true') {
+    const data = JSON.stringify({
+      project: 'terra-ui',
+      suite: suiteName,
+    });
+
+    try {
+      const req = http.request({
+        hostname: 'monstrous-screen-grabber.herokuapp.com',
```

Comment:
Some of this configuration is specific to us. Should this be broken out into a config file?